test/Widgets.test.yml

```
@@ -0,0 +1,166 @@
+  - wait-serial: "#LOG# WidgetList::nextValue=1"
+  - simulate: upButton-press
+  - wait-serial: "#LOG# WidgetList::nextValue=2"
+  - simulate: downButton-press
+  - wait-serial: "#LOG# WidgetList::previousValue=1"
+  - simulate: downButton-press
+  - wait-serial: "#LOG# WidgetList::previousValue=F"
+  - simulate: downButton-press
+  - wait-serial: "#LOG# WidgetList::previousValue=E"
+  - simulate: enterButton-press
+  - wait-serial: "54DE"
+  - wait-serial: "#LOG# ItemWidget::exitEditMode=Pin"
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)